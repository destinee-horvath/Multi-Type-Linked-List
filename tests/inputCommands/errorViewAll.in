view all
VieW aLL 345
VIEWALL
viewall
VIEW ALL 10