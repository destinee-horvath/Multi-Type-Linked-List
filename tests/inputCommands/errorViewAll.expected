INVALID COMMAND: INPUT
INVALID COMMAND: INPUT
INVALID COMMAND: VIEW
INVALID COMMAND: INPUT
INVALID COMMAND: VIEW ALL
